qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -57,26 +54,10 @@ def run_logstash_instance(config_name, options = {}, &block)
                                     "--path.config", config_to_temp_file(@fixture.config(config_name, options)),
                                     "--path.data", get_temp_path_dir,
                                     "--api.http.port", api_port.to_s)
-    wait_for_logstash(logstash_service)
-
+    logstash_service.wait_for_rest_api
```

Comment:
It seems we need to wait for the port before rest API.
```suggestion
    logstash_service.wait_for_logstash
    logstash_service.wait_for_rest_api
```

Reply:
🤔 why would we need to send `LogstashService#wait_for_logstash` twice in a row?

Reply:
1st method is `wait_for_logstash` and 2nd is `wait_for_rest_api`. `wait_for_rest_api` doesn't `rescue` connection refusal, so I thought `wait_for_logstash` will resolve. I will modify the logic to use single woit for rest API.
`wait_for_logstash` is 1) not necessary, 2) doesn't use the port we generate here;

FYI: once artifact API 503 is resolved I will test on my VM (just fired up new one) and suggest the working version.